windowPreview.js

```diff
@@ -359,7 +359,7 @@ class WindowPreviewMenuItem extends PopupMenu.PopupBaseMenuItem {
                 ? Clutter.ActorAlign.START : Clutter.ActorAlign.END,
             y_align: Clutter.ActorAlign.START,
         });
-        this.closeButton.add_child(new St.Icon({icon_name: 'window-close-symbolic'}));
+        this.closeButton.set_child(new St.Icon({icon_name: 'window-close-symbolic'}));
```

Comment:
This commit should be squashed into the commit that it fixes, and a comment added explaining why `set_child` is needed (copy from 255ef10db9c330643188faca5ea82093f7ad0bb0).

Reply:
I did a cherry-pick with the original commit. But it isn't in master...